src/components/admin/AddCourse.vue

```diff
@@ -86,18 +79,20 @@ export default defineComponent({
 
 		const course: Ref<Course> = ref(newCourse());
 		const user: Ref<User> = ref(newUser());
-		const login: Ref<string> = ref('');
+		const username: Ref<string> = ref('');
 		const instructor_exists = ref(false);
-		const course_dates: Ref<DateRange> = ref({ to:'', from:'' });
+		const course_dates: Ref<DateRange> = ref({ to: '', from: '' });
 
 		return {
 			course,
 			user,
-			login,
+			username,
 			course_dates,
 			instructor_exists,
-			checkUser: async () => { // lookup the user by login to see if already exists
-				const _user = await store.dispatch('users/getGlobalUser', user.value.login) as User;
+			checkUser: async () => {
+				// lookup the user by username to see if already exists
+				const _user = (await store.dispatch('users/getGlobalUser', user.value.username)) as User;
+				console.log(_user);
```

Comment:
Delete all `console.log`'s before committing.  A general practice you should get used to is to run `npm run build` or `quasar build` before committing.  `console.log`'s will break the production build.

Reply:
Furthermore, the production build runs all linters with errors.  The development build does not.

Reply:
done.